digitalocean/import_digitalocean_monitor_alert_test.go

```diff
@@ -0,0 +1,56 @@
+package digitalocean
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccDigitalOceanMonitorAlert_importBasic(t *testing.T) {
+	// copied from the database import test, but 3/3 does not fail ...
+
+	randName := randomTestName()
+	resourceName := fmt.Sprintf("digitalocean_monitor_alert.%s", randName)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckDigitalOceanMonitorAlertDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(testAccAlertPolicy, randName, "", "10m", "v1/insights/droplet/memory_utilization_percent", "Alert about memory usage"),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccMonitorAlertImportID(resourceName),
+			},
```

Comment:
This part of the test fails.

Reply:
Removing `ImportStateIdFunc` was enough to get this working! I pushed a commit with that change. That function is only needed when the resource's ID is not enough to pass to the read function. For example, with a database user you need both the ID of the parent cluster and the username itself. In this case we can just pass in the alert's ID directly.